### PR TITLE
Runs an HL7 file, outputs JSON to console

### PR DESCRIPTION
**Pull request by LisaWellman**
Creates the tools folder, with 2 tools
- run the converter on any HL7 file on the file system, outputs JSON to System.out
- run the converter on all *.hl7 files in a folder, results written to output folder

Also removed a println writing debug info to the console.

Signed-off-by: Lisa S. Wellman <peace@us.ibm.com>

### CODE REVIEW

File: src/tools/io/github/linuxforhealth/hl7/message/tools/FHIRConverterRunFile.java [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+                .build();
+
+        String json = ftv.convert(new File(filename), options);
+        System.out.println("----------------");
+        System.out.println(json);
+        System.out.println("----------------");
+
+        int dot = filename.lastIndexOf('.');
+        int pathSep = filename.lastIndexOf(File.separator);
+        String path = "C:\\aaawork\\code\\temp files\\";
```
Comment (klwhaley) [inline src/tools/io/github/linuxforhealth/hl7/message/tools/FHIRConverterRunFile.java:61]: make output path a env variable, not your local system path

File: src/tools/io/github/linuxforhealth/hl7/message/tools/FHIRConverterRunFile.java [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+            String line = br.readLine();
+            while (line != null) {
+                sb.append(line);
+                sb.append(System.lineSeparator());
+                line = br.readLine();
+            }
+            everything = sb.toString();
+        } finally {
+            br.close();
+        }
```
Comment (klwhaley) [inline src/tools/io/github/linuxforhealth/hl7/message/tools/FHIRConverterRunFile.java:45]: this isn't used is it? line 54 using the file directly which is what i expected

**Review by klwhaley — APPROVED** [review]
Approved, but some changes needed, see comments

File: src/tools/io/github/linuxforhealth/hl7/message/tools/FHIRConverterRunFile.java (hunk 1 above)
Comment (LisaWellman) [reply src/tools/io/github/linuxforhealth/hl7/message/tools/FHIRConverterRunFile.java:61]: thank you, fixed

File: src/tools/io/github/linuxforhealth/hl7/message/tools/FHIRConverterRunFile.java (hunk 2 above)
Comment (LisaWellman) [reply src/tools/io/github/linuxforhealth/hl7/message/tools/FHIRConverterRunFile.java:45]: yes, looks like I missed some cleanup, fixed

**Pull request merged**